src/pages/workspaces/workspace/analysis/modals/AzureComputeModal.js

```diff
@@ -201,12 +191,16 @@ export const AzureComputeModalBase = ({
 
     //each branch of the cond should return a promise
     await Utils.cond(
-      [viewMode === 'deleteEnvironment',
-        () => Ajax().Runtimes.runtimeV2(workspaceId, currentRuntimeDetails.runtimeName).delete()], //delete runtime
+      [viewMode === 'deleteEnvironment', () => {
+        return Utils.cond(
```

Comment:
this doesn't need the brackets and return, `() => Utils.cond(...)` is sufficient